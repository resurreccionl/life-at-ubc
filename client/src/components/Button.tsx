import React from 'react';
import styled from 'styled-components';

const StyledButton = styled.div`
  height: 45px;
  margin-left: 20px;
  margin-right: 20px;
  border: 1px solid #747474;
  border-radius: 12px;
  text-align: center;
  line-height: 45px;
  cursor: pointer;
  flex: 1;
`;

interface ButtonProps {
  content: string | number;
}

function Button({ content }: ButtonProps) {
  return <StyledButton>{content}</StyledButton>;
}

export default Button;
