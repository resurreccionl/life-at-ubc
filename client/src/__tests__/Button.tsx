import React from "react";
import { mount } from "enzyme";
import Button from "../components/Button";
// import Enzyme from "enzyme";
// import Adapter from "enzyme-adapter-react-16";

// Enzyme.configure({ adapter: new Adapter() });

describe("Home", () => {
  it("renders properly", () => {
    const wrapper = mount(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
