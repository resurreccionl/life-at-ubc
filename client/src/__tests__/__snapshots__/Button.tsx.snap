// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button can be rendered properly 1`] = `ReactWrapper {}`;

exports[`Button renders properly 1`] = `ReactWrapper {}`;
